src/app/trainer/dashboard/clients/page.jsx

```
@@ -0,0 +1,484 @@
+            const client = clientRequest.client;
+            const profile = client.clientProfile;
+            const clientGender = profile?.gender;
+            const genderStyles = getGenderStyles(clientGender);
+            const isMale = clientGender?.toLowerCase() === "male";
+            const isFemale = clientGender?.toLowerCase() === "female";
+
+            return (
+              <div
+                key={clientRequest.id}
+                className={`relative group w-full ${genderStyles.background} border ${genderStyles.border} rounded-xl shadow-lg ${genderStyles.shadow} transition-all duration-300 hover:-translate-y-0.5 backdrop-blur-sm overflow-hidden cursor-pointer`}
+                role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)